snowfall/training/ctc_graph.py

```diff
@@ -0,0 +1,159 @@
+# Copyright (c)  2020  Xiaomi Corp.       (author: Fangjun Kuang)
+
+from functools import lru_cache
+from typing import (
+    Iterable,
+    List,
+)
+
+import torch
+import k2
+import _k2
+
+
+def build_ctc_topo(phones: List[int]) -> k2.Fsa:
```

Comment:
Would be more generic as: phones -> tokens

Reply:
Renamed to `tokens`.